### PR TITLE
Delete Open Policy Agent (OPA) integration

### LINKED ISSUE #35: Don't list all OPA policies on command line
Files:
- contrib/gardener/scripts/deploy-karydia-cp
- manifests-dev/karydia-dev-telepresence.yml

list all the OPA policies on the command line:
```
      - name: opa
        image: openpolicyagent/opa:0.10.1
        imagePullPolicy: IfNotPresent
        args:
          - "run"
          - "--server"
          - "--addr=http://127.0.0.1:8181"
          - "/etc/opa-policies/parameters.rego"
          - "/etc/opa-policies/default-policy-matches.rego"
          - "/etc/opa-policies/default-kubernetes-matches.rego"
...
```
because I thought it was not possible to avoid loading dotted files from configmaps.

But `opa` has the option `--ignore`:
https://github.com/open-policy-agent/opa/issues/1109#issuecomment-448395503

We should try this.

### LINKED ISSUE #134: Delete Open Policy Agent (OPA) integration
### Description
karydia does not make use of OPA anymore. Thus, the integration should be deleted. This includes all snippets in code and documentation. A (incomplete) list, where adjustments have to be made:
- [x] Rules
- [x] Start script
- [x] Namespace
- [x] Execution

### User Story
_As a user I want to have an easy overview of current and supported features of karydia in order to gain a brief understanding of karydia's purpose/features and the current development state._

### PR DESCRIPTION
**Pull request by Neumann-Nils**
### Description
As karydia does not make use of OPA anymore, the integration should be deleted. This includes all snippets in code, documentation and auto-generated dependencies.

Resolves #134.
Closes #35.

### Checklist
Before submitting this PR, please make sure:
- [x] your code builds clean with `make`
- [x] your code lets succeed unit tests with `make test`
- [x] your code lets succeed integration tests
- [x] you have documented new or changed features

### CODE REVIEW

File: contrib/gardener/manifests/rbac-cp.yml [hunk 1]
```diff
@@ -84,37 +84,7 @@ roleRef:
...
-  apiGroup: rbac.authorization.k8s.io
-roleRef:
-  kind: Role
-  name: configmap-modifier
-  apiGroup: rbac.authorization.k8s.io
-
----
-
-# Define role for OPA/kube-mgmt to cache the resources
+# Define role for kube-mgmt to cache the resources
```
Comment (SAPJonasZengerle) [inline contrib/gardener/manifests/rbac-cp.yml]: I think this kube-mgmt stuff was only related to OPA and, thus, can be deleted as well.

File: docs/features.md [hunk 2]
```diff
@@ -4,7 +4,6 @@
 |---------|-----------|---------------------------|-----------------------------------|--------|
```
Comment (SAPJonasZengerle) [inline docs/features.md]: At `docs/development-and-gardener-setup.md` there is still a mentioning of OPA policies which should be deleted as well.
```
(plus some additional configuration of demo OPA policies)
```

Comment (Neumann-Nils) [reply docs/features.md]: Done.

File: contrib/gardener/manifests/rbac-cp.yml (hunk 1 above)
Comment (SAPJonasZengerle) [reply contrib/gardener/manifests/rbac-cp.yml]: This point is still open.

Comment (Neumann-Nils) [reply contrib/gardener/manifests/rbac-cp.yml]: Should be deleted (see 325d71e8cd0e6c48ea81e1ebd7b16b400afa8889)

**Pull request merged**